test/objects.coffee

```diff
@@ -44,6 +44,15 @@ pluck = (x) -> x.a
 eq 100, pluck pluck pluck a: a: a: 100
 
 
+test "Object literal spread", ->
+  obj1 =
+    a: 1
+    b: 2
+  # obj2 = "#{{obj1..., b: 3}}"
+  t1 = CoffeeScript.compile "{obj1..., b: 3}", bare: yes
+  t1 = t1.replace(/\s+/g, "")
+  eq t1, "({...obj1,b:3});"
```

Comment:
The rule has been to never compare strings with the compiled output JS, unless there's no other way to assert that the compiled code works as intended. Also, CoffeeScript always strives to produce readable output. You need to add some spaces in there.